static/scripts/onboarding/onboarding.ts

```diff
@@ -131,24 +49,24 @@ export const parseYAML = async (data: any): Promise<any | undefined> => {
   }
 };
 
-export const parseJSON = async (data: any): Promise<any | undefined> => {
+export const parseJSON = async <T>(data: string) => {
   try {
     const parsedData = await JSON.parse(data);
-    return parsedData;
+    return parsedData as T;
   } catch (error) {
     return undefined;
   }
 };
 
 export const YAMLStringify = (value: any) => YAML.stringify(value, { defaultKeyType: "PLAIN", defaultStringType: "QUOTE_DOUBLE", lineWidth: 0 });
 
-export const getConf = async (initial: boolean = false): Promise<string | undefined> => {
+export const getConf = async (): Promise<string | undefined> => {
```

Comment:
I guess I need to update the lint settings after this is merged. I banned these arrow functions in favor of named functions (unless they are in a callback.)